### PR TITLE
fix: Only update seekBar during pause when live

### PR DESCRIPTION
**Pull request by brandonocasey**
## Description
During live playback the seek bar only has to be updated when the player is paused, because we will always be the same period away from live while playing. 

## Specific Changes proposed
* Start updating the seekbar for the first time on duration change.
* If the duration is Infinity only update when paused otherwise update how we do now.
* Make sure visibility change only enables the interval if it was enabled before losing focus.

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/control-bar/progress-control/seek-bar.js [hunk 1]
```diff
@@ -40,62 +40,92 @@ class SeekBar extends Slider {
    */
   constructor(player, options) {
     super(player, options);
-    this.setEventHandlers_();
+    this.update_ = Fn.bind(this, this.update);
```
Comment (brandonocasey) [inline src/js/control-bar/progress-control/seek-bar.js]: Was kind of silly to rebind these in a helper, and we need them in both `doEvent` functions.

**Comment by gkatsev** [comment]
This change makes sense over-all but it sort of introduces the same bug that https://github.com/videojs/video.js/pull/6445 fixed where the current time tooltip doesn't update but it does if you move the mouse which looks a bit weird since when you just hover nothing changes but then if you move the mouse a bit the current time tooltip could update.
Tested via https://deploy-preview-6413--videojs-docs.netlify.com/test-example/liveui.html.
Maybe we should always update the tooltips but we can stop everything else? Maybe that change is fine? It's just a second or two. Maybe this is something that should be fixed/handled by VHS?

**Comment by gkatsev** [comment]
Looks like maybe the issue is further reduced in #6405, so, may be less of a problem with both changes in.

**Comment by brandonocasey** [comment]
We can revisit his one once the other pull request is merged I would think.

**Comment by brandonocasey** [comment]
Going to close this, I don't think that it's worth doing.

**Pull request closed without merging**